### PR TITLE
add Django 3.0 to the test matrix

### PR DESCRIPTION
**Pull request by mvbrn**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -20,6 +21,7 @@ deps =
 	django20: Django>=2.0,<2.1
 	django21: Django>=2.1,<2.2
 	django22: Django>=2.2,<3.0
+	django30: Django>=3.0a1,<3.1
```
Comment (jschneier) [inline tox.ini]: Can you put `< 4.0` please.

File: setup.py [hunk 2]
```diff
@@ -37,6 +37,7 @@ def read(filename):
         'Framework :: Django :: 2.0',
         'Framework :: Django :: 2.1',
         'Framework :: Django :: 2.2',
+        'Framework :: Django :: 3.0',
```
Comment (jschneier) [inline setup.py]: I requested a Trove classifier, it hasn't been added yet.

https://github.com/pypa/warehouse/issues/6605

Comment (mvbrn) [reply setup.py]: Should I remove that line for now?

Comment (jschneier) [reply setup.py]: You can remove it or comment it out,  either way.

**Pull request merged**